Neo4j.Driver/Neo4j.Driver/ConfigBuilder.cs

```diff
@@ -343,7 +343,8 @@ public ConfigBuilder WithUserAgent(string userAgent)
 
     /// <summary>
     /// Override configuration for which <see cref="INotification"/>s should be emitted for the lifetime of the
-    /// driver. <br/> Unspecified configuration will be provided by configuration in the server.
+    /// driver. <br/> Unspecified configuration will be provided by configuration in the server. <br/>
+    /// Disabling categories or severities allows the server to skip analysis for those, which can speed up query execution.
```

Comment:
> Disabling categories or severities allows the server to skip analysis for those

suggest adding text to make it "for those categories or severities" since I had to read a couple of times to get it.